### DISCUSSION TITLE
Remove TS logo.

### DISCUSSION THREAD

**Discussion by jimmywarting** [npm/feedback#743] [Website]
I found somebody posting this on another repo today

> ![image](https://user-images.githubusercontent.com/1148376/174257172-6dbe1b35-67a7-4173-9a0a-1ca1947bc5a0.png)

ppl are in disbelieve that a vanilla javascript can be as equally good typed along with JSDoc.
I use type safety all the time, i even have checkJS, allowJS and strict type checking on, but I never write anything in typescript flavored syntax. 

typescript is able to parse javascript code just fine and give type hinting just as regular JSDoc

it gives ppl like me who do not write in code in typescript an unfair disadvantage

**Comment by ljharb** [comment]
You don’t have to author code in TS; i believe it’s sufficient for there to be a DT package in existence.

**Reply by jimmywarting** [reply]
but not all code needs to have DT, specially so if it's written with jsdoc.

**Reply by ljharb** [reply]
and you’re unwilling to generate a d.ts file from the jsdoc?

**Reply by jimmywarting** [reply]
again, all code don't need d.ts if typescript can scan js files anyway

**Reply by GabenGar** [reply]
On other hand when I click on a symbol definition I don't want to see recursive abstract factory cache decorator calls when the only thing I care about is what single-letter generics mean.

**Reply by jimmywarting** [reply]
I have used the Deno plugin for vs code "to replace vscode own typescript parser" and imported something with HTTP import. the javascript code was full of jsdoc code all over the place and it gave me just as good type hinting as regular d.ts if not even better if it had been generating a d.ts. What i like about this even more is that i can click to show definitions and also see the source code of what it actually dose. 
when things are written with jsdoc instead it tends to be more explanatory then as most TS user tends to avoid writing jsdoc comments at all...

**Comment by wesleytodd** [comment]
I feel like this conversation is similar to the "showing tarball size" in some ways but with a distinct difference. The presence of typings for TS users is basically a must. If a user feels that way, then let them. Even if they are wrong, there are a lot more wrong things folks believe which are more harmful than this.  Compare this to the tarball size, which gives a wrong indication of size but also has no meaningful upside to a part of the userbase and was removed.

IMO we should make it easy and intuitive for folks no matter their chosen tool set to find and consume packages, and removing this will hurt TS users and not meaningfully help everyone else. Additionally, "advantage" goes to the maintainer who is willing to help ALL users, so I am not sure it holds that folks not authoring in TS are disadvantaged considering @ljharb's point above that all you need to do is generate a `.d.ts` file.

PS. I also do not author TS, and am generally not a fan, so this is not coming from the place of being a fan.

**Reply by ljharb** [reply]
Exactly - tarball size causes FUD; having TS types or not doesn’t (there’s no guarantee they’re correct even if they exist).
